### ISSUE TITLE
Type mismatch on ParticipantChangedEventModel

### ISSUE THREAD

**Issue by phaalonso** [open-wa/wa-automate-nodejs#2036]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Current Behavior

In the ParticipantChanged event, the typing of the event don't correspond to the one being send to it.

While it's says that it will be of the type Id, it is receiving strings. Furthermore, making the need to cast the variable to unknown and then the type needed in the function, like this:
```ts
this.client.onGlobalParticipantsChanged(async change => {
    if (change.action != 'add') return;

    let newMembers: Id[];

    if (Array.isArray(change.who))
        newMembers = [...change.who];
    else
        newMembers = [change.who];
    }

    console.log(newMembers);
    newMembers.forEach(userId => {
        const id = userId.toString() as ContactId;
        if (!id.startsWith('55')) {
            client.getChatById(change.chat as unknown as `${number}@c.us`).then(chat => {
                if (chat.canSend) {
                    client.sendText(chatId, 'Esse grupo é restrito para usuários pertencentes do Brasil');
                }
            });
            client.removeParticipant(change.chat as unknown as GroupChatId, id)
                .catch(err => logger.exception(err, 'Remove +55 from group'));
        } else {
            verifyBlackList(id);
        }
    });
});
```

Without casting, this error will happen:
```
Argument of type 'Id' is not assignable to parameter of type '`${number}@c.us`'.
```

Type used in typescript:
```ts
export interface ParticipantChangedEventModel {
  by: Id,
  action: groupChangeEvent,
  who: [Id]
  chat: Id
}

export interface Id {
  server: string;
  user: string;
  _serialized: string;
}
```

### Expected Behavior

The types of ParticipantEvent be equals to the ones send to the user, and is expected that they are compatible with the clients functions 

### Steps To Reproduce

1. In a normal bot, use `client.onParticipantChanged` or `client.onGlobalParticipantsChanged`
2. Log the variable used in the callback
3. Check if type send to the callback is the same as the type that Typescript knows
3. Check if it can be used in others functions

### Mode

My own code

### create() code

```markdown
create({
    headless: false,
    killProcessOnBrowserClose: true,
    killClientOnLogout: true,
})
```

### DEBUG INFO

```markdown
WA_VERSION: 2.2132.6                                                                         
PAGE_UA: 'WhatsApp/2.2108.8 Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36' │                                           
BROWSER_VERSION 'Chrome/93.0.4577.0'
```

### Environment

```markdown
- OS: Pop!_OS 21.04 x86_64 
- Node: v15.14.0
- npm: 7.7.6
```

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.17.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.17.1...4.17.2) (2021-08-23)

- Release 4.17.2 [`f152937`](https://github.com/open-wa/wa-automate-nodejs/commit/f152937a64152b96056cf92703546c974c5731df)
- 🏷️ fix `ParticipantChangedEventModel` type #2036 [`b721e00`](https://github.com/open-wa/wa-automate-nodejs/commit/b721e001cb5ff963332f82b09c3446f6a942ed44)
- 🔧 fix groupmetadata imports [`ed4b77e`](https://github.com/open-wa/wa-automate-nodejs/commit/ed4b77eb6dc98b476d3cc13572dad6d0d7b1c719)
